website/docs/docs/building-a-dbt-project/package-management.md

```diff
@@ -108,33 +108,78 @@ To find the latest release for a package, navigate to the `Releases` tab in the
 
 As of v0.14.0, dbt will warn you if you install a package using the `git` syntax without specifying a version (see below).
 
-#### Private packages
-Private packages can be installed by using the SSH configuration on the machine or by passing in a username and password with the git URL. It is recommended that you use SSH authentication method where possible.
+### Private packages
 
+#### SSH Key Method
+Private packages can be installed by using the SSH configuration on the machine. This method requires the SSH configuration be stored in `~/.ssh/`.
```

Comment:
This reads a little funny - is this a place where it makes sense to link out to docs like:
 - https://docs.github.com/en/github/authenticating-to-github/connecting-to-github-with-ssh
 - https://docs.gitlab.com/ee/ssh/

?

Reply:
Sounds good I'll update it. I tried to preserve the original language used in the docs, but it wasn't that clear to me either.